openedx_lti_tool_plugin/deep_linking/api/v1/views.py

```diff
@@ -0,0 +1,49 @@
+"""Django Views."""
+from django.db.models import QuerySet
+from rest_framework.mixins import ListModelMixin
+from rest_framework.pagination import PageNumberPagination
+
+from openedx_lti_tool_plugin.deep_linking.api.v1.serializers import CourseContentItemSerializer
+from openedx_lti_tool_plugin.deep_linking.api.views import DeepLinkingViewSet
+from openedx_lti_tool_plugin.models import CourseContext
+from openedx_lti_tool_plugin.utils import get_identity_claims
+
+
+class ContentItemPagination(PageNumberPagination):
+    """Content Item Pagination."""
+
+    page_size_query_param = 'page_size'
+    max_page_size = 100
+
+
+class CourseContentItemViewSet(
+    ListModelMixin,
+    DeepLinkingViewSet,
+):
+    """Course Content Item ViewSet.
+
+    A content item is a JSON that represents any content the LTI Platform can consume,
+    a content item could be an LTI resource link launch URL, a URL to a resource hosted
+    on the internet, an HTML fragment, or any other kind of content type.
+
+    This ViewSet returns a list of LTI Resource Link content items for each Course
+    available for the LtiTool related to the request launch data and the
+    site configuration `course_org_filter` setting.
+
+    """
+
+    serializer_class = CourseContentItemSerializer
+    pagination_class = ContentItemPagination
+
+    def get_queryset(self) -> QuerySet:
+        """Get QuerySet.
+
+        Returns:
+            CourseContext QuerySet.
+
+        """
+        iss, aud, _sub, _pii = get_identity_claims(self.launch_data)
```

Comment:
@kuipumu Please explain what is `iss, aud, _sub, _pii.`

Reply:
@anfbermudezme The get_identity_claims function has a docstring with references to the OpenID Connect Core 1.0 ID token claims and standard claims, I will add a comment in this line to explain why we are obtaining the iss and aud claim